roles/apache/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: install apache package
+  apt: pkg=apache2
+
+- name: install apache module packages
+  apt: pkg={{ item }}
+  with_items: apache.modules
+
+- name: enable apache modules
+  apache2_module: name={{ item }}
+  with_items:
+    - headers
+    - ssl
+    - rewrite
+    - uwsgi
+  notify: reload apache
+
+- name: disable default vhost
+  file: dest=/etc/apache2/sites-enabled/{{ item }}
+        state=absent
+  with_items:
+    - 000-default
+    - 000-default.conf
+    - default
+    - default.conf
+  notify: stop apache
```

Comment:
I can fix this to use the module. I really didn't look too much in depth at the apache play book, I pretty much just copied it over. I'll give the whole play book a more thorough review. and re-submit the changes.